src/cli/node/with_node.rs

```diff
@@ -235,6 +259,20 @@ pub enum Command {
         #[structopt(long = "limit", allow_hyphen_values = true, default_value = "-50")]
         limit: i64,
     },
+    Claim {
```

Comment:
```suggestion
#[structopt(
        name = "claim",
        about = "Claim identity"
    )]
    Claim {
```

Reply:
I think that adding this lines gives a description to the command. At the moment when writing --help it appears an empty line 
![image](https://user-images.githubusercontent.com/52961116/90111720-54052680-dd4f-11ea-8d5b-2a8b989528fc.png)

Reply:
Good one! I completely forgot it!

Reply:
I wrote this: "Claim a Witnet identity by signing the identifier with the node master key"